app/assets/stylesheets/application.css

```diff
@@ -1,15 +0,0 @@
-/*
```

Comment:
You should not remove file, although it only contain comments

Reply:
oh i see, because the file has the same content like file application.scss. so when I run localhost it gives error.